src/baked-potatoes.md

```diff
@@ -0,0 +1,29 @@
+# Baked Potatoes
+
+- ⏲️ Prep time: 10 minutes
+- 🍳 Cook time: 20-25 minutes
+- 🍽️ Servings: 4-6 people
+
+## Ingredients
+
+- 6-7 Yukon pototoes (you can use any potatoes you like)
+- 1/3 cup of any vegitable oil
+- 1 teaspoon salt
+- Black pepper
+- Any seasoning you like
```

Comment:
```suggestion
```

Remove salt/pepper and seasonings 'to taste' from ingredients. They can be mentioned below in the directions.